ui/src/wizard/StorageAccountListStep.ts

```diff
@@ -77,15 +79,21 @@ export class StorageAccountListStep<T extends IStorageAccountWizardContext> exte
             const client: StorageManagementClient = new StorageManagementClient(wizardContext.credentials, wizardContext.subscriptionId);
 
             const quickPickOptions: IAzureQuickPickOptions = { placeHolder: 'Select a storage account.', id: `StorageAccountListStep/${wizardContext.subscriptionId}` };
-            wizardContext.storageAccount = (await ui.showQuickPick(this.getQuickPicks(client.storageAccounts.list()), quickPickOptions)).data;
+            const result: StorageAccount | undefined | string = (await ui.showQuickPick(this.getQuickPicks(client.storageAccounts.list()), quickPickOptions)).data;
+            if (isString(result)) {
```

Comment:
I'm assuming this is to the open the "Learn more..." page, but I feel like the conditional feels very vague.  Maybe a comment would be clarifying?

Reply:
Also, do we necessarily want to close the quick pick menu after opening a new page for the user?  I feel like it'd be more helpful to keep the quick pick list active so that when they navigate back to VS Code, they can resume where they left off rather than having to repeat whatever initial steps.

Reply:
Yeah I thought about doing that, but eventually decided it wasn't worth the effort/code-complexity. I'll fix in a separate PR if you think it's worth it

Reply:
Definitely not blocking, but could you make a P3 issue on it? I can see this being a minor annoyance.

Reply:
https://github.com/Microsoft/vscode-azuretools/issues/176